Makefile

```diff
@@ -3,6 +3,14 @@ TESTDIR ?= "prime/"
 SNAP_NAME=core24
 BUILDDIR=/build/$(SNAP_NAME)
 
+# include any fips environmental setup if the file exists.
+# Variables:
+# - SNAP_FIPS_BUILD
+ifneq (,$(wildcard ./.fips-env))
+    include .fips-env
```

Comment:
Instead of using a if, maybe just `-include`, with the dash as prefix.

Reply:
Can we keep the `ifneq` ? It feels more explicit that this file is optional, and then we don't try to export a variable that might not exist

Reply:
What about `ifdef SNAP_FIPS_BUILD` to guard the export? That is easier to read than `ifneq (,$(wildcard ./.fips-env))`.

Reply:
I'd say that is okay as long as we only export one variable, otherwise it quickly gets very verbose